ChangeLog

```diff
@@ -1,3 +1,10 @@
+2015-01-13  Daniel Standage <daniel.standage@gmail.com>
```

Comment:
This should be merged with the other entry below. Two spaces after the name and before the bracketed email address please.